add Description texts & error messages styling

Makisai opened the pull request:
- Backend: add final texts to seeds
- Frontend: add description texts with locals to walksDetail
- add styling to error and success texts
- add css class for buttons
- add titel/region/length to walksDetail
- delete NewFriendsButton

Makisai committed:
add Description texts & error messages styling
- Backend: add final texts to seeds
- Frontend: add description texts with locals to walksDetail
- add styling to error and success texts
- add css class for buttons
- add titel/region/length to walksDetail
- delete NewFriendsButton

MichelleWeck commented on Frontend/src/components/PhotoUpload.vue:
```diff
@@ -46,8 +51,8 @@
       challengeIsSelected() {
-        const challengeIds = this.$store.state.detail.photowalk.challenges.map(challenge => challenge.id);
-        return challengeIds.includes(this.$store.state.detail.selectedChallenge.id);
+          const challengeIds = this.$store.state.detail.photowalk.challenges.map(challenge => challenge.id);
+          return challengeIds.includes(this.$store.state.detail.selectedChallenge.id);
```
Hier Einrückung, ist aber nicht notwenig

MichelleWeck commented on Frontend/src/locales/en.json:
```diff
@@ -81,17 +81,21 @@
+    "lastPhotowalk": "This was your last Photowalk",
     "tutorial": "How does Photowalk work?",
-    "navigate":"Take a Photowalk.",
+    "navigate":"Take a Photowalk",
```
Hier vielleicht "Go on a Photowalk"?

Makisai replied:
Ich hatte gedacht take, weil man ja auch sagt "take a walk"

MichelleWeck replied:
Ja das stimmt, ich dachte aber wenn man neu zu Photowalk ist, ist go on a PhotoWalk eindeutiger, das du irgendwo hin gehst und dir nicht nen Photowalk nimmst? Ist wohl subjektiv, wir können es auch so lassen

Mike10101 replied:
Lmao. ursprünglich wars auch Go on a Photowalk. Naja mir egal.

MichelleWeck commented on Frontend/src/views/WalksDetail.vue:
```diff
@@ -105,6 +124,9 @@ export default {
     }).then(response => {
       this.$store.commit(SET_PHOTOWALK,response.data);
       this.isLoading = false;
+      this.photowalkTitle = response.data.name;
+      this.photowalkRegion = response.data.region;
+      this.photowalkLength = response.data.length;
```
Warum werden diese drei Datensätze extra abgespeichert? Die werden doch vorher ebenfalls in den Store commited und können daraus ausgelesen werden?

MichelleWeck commented on Frontend/src/views/WalksDetail.vue:
```diff
@@ -8,6 +8,14 @@
             <PhotowalkMap/>
           </v-col>
           <v-col cols="12" md="6">
+              <h3 class="text-h3" v-text="photowalkTitle"/>
+              <h4 class="text-h4" v-text="photowalkRegion"/>
+              <span v-text="photowalkLength"/>
```
Warum nicht aus dem Store?

MichelleWeck commented on Frontend/src/views/WalksDetail.vue:
```diff
@@ -49,6 +57,14 @@
           </v-col>
         </v-tab-item>
         <v-tab-item>
+          <h3 class="text-h3" v-text="photowalkTitle"/>
+          <h4 class="text-h4" v-text="photowalkRegion"/>
+          <span v-text="photowalkLength"/>
```
Warum nicht aus dem Store?

MichelleWeck commented on Frontend/src/views/WalksDetail.vue:
```diff
@@ -49,6 +57,14 @@
+          <h4 class="text-h4" v-text="photowalkRegion"/>
+          <span v-text="photowalkLength"/>
+          <v-row>
+            <p v-if="this.$store.state.detail.currentID === 1">{{$t('photos.description1')}}</p>
+            <p v-if="this.$store.state.detail.currentID === 2">{{$t('photos.description2')}}</p>
+            <p v-if="this.$store.state.detail.currentID === 3">{{$t('photos.description3')}}</p>
```
Kann man wie oben abkürzen

MichelleWeck commented on Frontend/src/views/WalksDetail.vue:
```diff
@@ -92,7 +108,10 @@ export default {
       isLoadingMyPhotos: true,
-      isLoadingFriendsPhotos: true
+      isLoadingFriendsPhotos: true,
+      photowalkTitle:'',
+      photowalkRegion:'',
+      photowalkLength:0,
```
nötig?

MichelleWeck reviewed (commented):
Ich hab ein paar Anmerkungen zu WalksDetail, der Rest sieht gut aus

MichelleWeck commented on Frontend/src/views/WalksDetail.vue:
```diff
@@ -8,6 +8,14 @@
+              <h4 class="text-h4" v-text="photowalkRegion"/>
+              <span v-text="photowalkLength"/>
+            <v-row>
+              <p v-if="this.$store.state.detail.currentID === 1">{{$t('photos.description1')}}</p>
+              <p v-if="this.$store.state.detail.currentID === 2">{{$t('photos.description2')}}</p>
+              <p v-if="this.$store.state.detail.currentID === 3">{{$t('photos.description3')}}</p>
```
Lässt sich abkürzen: 
<p>{{$t('photos.description'+ this.$store.state.detail.currentID)}}</p>

MichelleWeck reviewed (commented):
Fehlende Anmerkung

Makisai committed:
Refactor for CR#37
- get photowalk informations from store
- shorten code to show walk description
- add bottom margin to photogrid on Landing Page

MichelleWeck reviewed (approved):
Alles klar, kann jetzt so gemergt werden
